content/en/blog/2020/workload-entry/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Introducing Workload Entries
+subtitle: Bridging Kubernetes and VMs
+description: Describing the new functionality of Workload Entries.
+publishdate: 2020-05-14
```

Comment:
@SecurityInsanity you should probably update this date, or the article will look old on the day it goes live.

```suggestion
publishdate: 2020-05-21
```

Reply:
Updated.